backend/envconfig/envconfig.go

```diff
@@ -81,3 +83,14 @@ func setFieldValue(field reflect.StructField, fieldValue reflect.Value, newValue
 func NewEnvConfig(environment fw.Environment) EnvConfig {
 	return EnvConfig{environment: environment}
 }
+
+func processInt(value, kind string) (int64, error) {
```

Comment:
```suggestion
func processInt(newValue, type reflect.Type) (int64, error) {
```
I think we need to ensure this duration is the official one define inside built-in `time` package.
`PkgPath` will help with this.

Reply:
Re thought about this. How about renaming to `parseInt`? `ProcessInt` didn't provide reader enough information.

Reply:
True, parseInt gives more insights.